### ISSUE TITLE
http://0.0.0.0:8000/chart.html

### ISSUE THREAD

**Issue by ChezPolo** [jonromero/forex_algotrading#14]
Edit 16/11/17: Issue not resolved. I was able to run the code till the end (verified by "Done. Goto 0.0.0.0:8000/chart.html"), and went into http://MYPCNAME:8000 but I could not find any "Chart" file in the server... Help from anyone??

**Comment by Brickly1** [comment]
Same issue here
 How was this resolved

**Comment by ChezPolo** [comment]
I got a feeling the .json file created can be or was meant to feed into a html interactive chart, but i don't think  that process is automated... gotta find a way to make it work

**Comment by ChezPolo** [comment]
yeah ok so a silly error really, chart.html file is in the ml folder but my dumbass didn't see it. However I'm trying to load the file but no chart appears so I'm trying to work that out.

**Comment by spsteady** [comment]
Did either of you figure out how to get the chart to display?  I am just getting a white page when it loads.
